### PR TITLE
Blog post for JP

### PR DESCRIPTION
**Pull request by odow**
cc @mlubin
cc @ccoffrin
cc @chkwon
cc @jajhall
cc @juan-pablo-vielma

### CODE REVIEW

File: pages/governance.md [hunk 1]
```diff
@@ -109,16 +109,20 @@ Steering Committee:
 The current members of the Steering Committee are:
 
 * Miles Lubin, _Hudson River Trading_ (@mlubin)
-* Juan Pablo Vielma, _Google_ (@juan-pablo-vielma)
 * Carleton Coffrin, _Los Alamos National Laboratory_ (@ccoffrin)
 * Oscar Dowson (@odow)
 * Changhyun Kwon, _KAIST_ (@chkwon)
+* Julian Hall, _Unniversity of Edinburgh_ (@jajhall)
```
Comment (odow) [inline pages/governance.md:115]:
```suggestion
* Julian Hall, _University of Edinburgh_ (@jajhall)
```

File: _posts/2024-07-30-julian.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+the [NSF funding](/announcements/2021/02/22/agreement/) that has funded the
+majority of JuMP maintenance over the last three years. In addition to his
+non-code contributions, Juan Pablo also ended up being the author of many of the
+early `BinaryBuilder` packages that enabled JuMP to distribute precompiled
+open-source binaries to users. Juan Pablo now works at Google, where he manages
+the team building [MathOpt](https://developers.google.com/optimization/math_opt),
+a new interface in OR-Tools that is heavily inspired by JuMP.
+
+Following our [Governance](/pages/governance) procedures, the remaining Steering
+Commitee members have invited Julian Hall (@jajhall) to join the Steering
```
Comment (odow) [inline _posts/2024-07-30-julian.md:27]:
```suggestion
Committee members have invited Julian Hall (@jajhall) to join the Steering
```

**Comment by mlubin** [comment]
LGTM but @juan-pablo-vielma should approve the blog post text. Maybe separate that out from the update to the list of committee members.

**Comment by odow** [comment]
Updated the list in https://github.com/jump-dev/jump-dev.github.io/pull/150

File: _posts/2024-07-30-julian.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+early `BinaryBuilder` packages that enabled JuMP to distribute precompiled
+open-source binaries to users. Juan Pablo now works at Google, where he manages
+the team building [MathOpt](https://developers.google.com/optimization/math_opt),
+a new interface in OR-Tools that is heavily inspired by JuMP.
+
+Following our [Governance](/pages/governance) procedures, the remaining Steering
+Committee members have invited Julian Hall (@jajhall) to join the Steering
+Committee.
+
+Julian is the lead developer of [HiGHS](www.highs.dev), the worlds fastest
```
Comment (blegat) [inline _posts/2024-07-30-julian.md:30]:
```suggestion
Julian is the lead developer of [HiGHS](www.highs.dev), the world's fastest
```

File: _posts/2024-07-30-julian.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Recently, Juan Pablo Vielma notified the [JuMP Steering Committee](/pages/governance/#steering-committee)
+that he is stepping down (effective immediately) from his role on the Steering
+Committee.
+
+Juan Pablo was instrumental in the success of JuMP, supervising Miles Lubin and
+Joey Huchette as Ph.D. students at MIT. Juan Pablo helped organize the first
+three iterations of the JuMP-dev workshop: [JuMP-dev 2017](/meetings/mit2017/) at
+MIT, [JuMP-dev 2018](/meetings/bordeaux2018/) in Bordeaux, and [JuMP-dev 2019](/meetings/santiago2019/)
+in Santiago, Chile. More recently, Juan Pablo was the Principal Investigator of
```
Comment (juan-pablo-vielma) [inline _posts/2024-07-30-julian.md:17]: I would change "the Principal Investigator" to "a Principal Investigator" (as technically Alan is the PI and I am co-PI).

**Review by juan-pablo-vielma — APPROVED** [review]

File: _posts/2024-07-30-julian.md (hunk 4 above)
Comment (odow) [inline _posts/2024-07-30-julian.md:17]:
```suggestion
in Santiago, Chile. More recently, Juan Pablo was a Principal Investigator of
```

**Pull request merged**